specs/protocol/configurability.md

```diff
@@ -441,4 +459,11 @@ configuration of the permissioned dispute game.<br/>
 **Requirement:** No requirement<br/>
 **Notes:** <br/>
 
+### Operator Fee Manager
+
+**Description:** Account authorized to modify the operator fee scalar. <br/>
+**Administrator:** [System Config Owner](#admin-roles)<br/>
+**Requirement:** `address(0)`<br/>
```

Comment:
Looking at this again, the requirement of `address(0)` makes no sense. We cannot require the system config owner to be null. There currently is no such thing as the operator fee manager in the code

```solidity
    function setOperatorFeeScalars(uint32 _operatorFeeScalar, uint64 _operatorFeeConstant) external onlyOwner {
        _setOperatorFeeScalars(_operatorFeeScalar, _operatorFeeConstant);
    }
```

We should delete this section and just make the above 2 be managed by the System Config Owner

Reply:
The idea here is that standard chains cannot have an operator fee manager. Maybe the "Requirement" description is a bit misleading here.

But I agree that the operator fee manager isn't super necessary as a role. I can't imagine many situations where you'd need it to be distinct from the system config owner